nativelib/src/main/scala/java/lang/reflect/Field.scala

```diff
@@ -0,0 +1,5 @@
+package java.lang.reflect
+
+class Field {
+  def get(obj: Object): Object = null
```

Comment:
The same problem as for `Class::getField`, it should throw via `???` given that it's not implemented.